test/end-to-end-tests/src/usecases/create-room.js

```diff
@@ -48,4 +48,39 @@ async function createRoom(session, roomName, encrypted=false) {
     session.log.done();
 }
 
-module.exports = {openRoomDirectory, createRoom};
+async function createDm(session, invitees) {
+    session.log.step(`creates DM with ${JSON.stringify(invitees)}`);
+
+    const roomListHeaders = await session.queryAll('.mx_RoomSubList_labelContainer');
+    const roomListHeaderLabels = await Promise.all(roomListHeaders.map(h => session.innerText(h)));
+    const dmsIndex = roomListHeaderLabels.findIndex(l => l.toLowerCase().includes('direct messages'));
+    if (dmsIndex === -1) {
+        throw new Error("could not find room list section that contains 'direct messages' in header");
+    }
+    const dmsHeader = roomListHeaders[dmsIndex];
+    const addRoomButton = await dmsHeader.$(".mx_RoomSubList_addRoom");
```

Comment:
for my sake, can we call this the startChat button? to not be confused with the other addRoom button which actually creates rooms

Reply:
oh, yes. We definitely need to do that.

Unfortunately the class is still weird, but the copy/paste fail can be fixed.